lib/Lec6-BinaryHeap/binaryHeap.js

```diff
@@ -0,0 +1,41 @@
+let array_length;
+let heap_root = (input, i) => {
+    let left = 2 * i + 1;
+    let right = 2 * i + 2;
+    let maxValue = i;
```

Comment:
Here is camel-case instead of snake-case used before. Please rewrite variables to one style